### ISSUE TITLE
[Suggestion] Improve custom ruleset configuration

### ISSUE THREAD

**Issue by waredrive** [BG3-RAW-Mods/bg3-RAW-addon#8]
Let me start by thanking you for the great work you do. I have a suggestion, if possible, to make the custom ruleset option more configurable by either splitting the UI changes (described as "RAW changes") from the game mechanic changes (described as "Other changes") or making the mechanic changes configurable. It would be nice to be able to fine-tune those values in the json file.

**Comment by arnaudpourbaix** [comment]
Could you detail a little more ?

I can't make configurable because this is a XML file. I have already split this PAK into 2 in the next version.
First PAK is RAW:
- Hide NPC health
- Hide failed perception rolls
- Hide passive rolls in dialogues
- Hide difficulty class during rolls

Other is my own settings:
- Enemy aggression: Tactician
- Camp cost multiplier: 1.5
- Trader price multiplier: 3

**Comment by waredrive** [comment]
Sounds good. Thanks.

**Comment by waredrive** [comment]
Sorry, I missed that you wanted more details. It's my first playthrough, so I don't want to go 'all in' with modding. I still find some of the core aspects of the game annoying, and I see that your custom rules (the UI part) address them. I don't want to change the camp cost, trader price, etc., just yet, and would like to only use the UI changes. It seems this will be addressed by you soon, so I'm happy with that. I just have a further question. If I only want to use the custom rules, do I need to install only the 'RAW-addon-ruleset.pak' and nothing else?

**Comment by arnaudpourbaix** [comment]
Please note that you can just not install this optional PAK and just play custom game. Every options I have changed can be set in custom game settings.

**Comment by waredrive** [comment]
I'm a bit confused now. :) So do you mean that the "RAW-addon.pak" has everything else in it, including all the other packs that are in the repo ("RAW-addon-druid.pak", "RAW-addon-monk.pak" and "RAW-addon-ruleset.pak")? So I only need the "RAW-addon.pak" and configure it through the json file to have the functionality of the other packs?

**Comment by arnaudpourbaix** [comment]
No, I need to update README about that. RAW-addon doesn't contain druid, monk, or ruleset

**Comment by waredrive** [comment]
Ok, so which packs do I need to install if I want to use the UI part of the custom rules?

**Comment by arnaudpourbaix** [comment]
It is not released yet, probably in a week.
